<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathPaint_QtClass</class>
 <widget class="QMainWindow" name="MathPaint_QtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>708</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MathPaint_Qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="imageBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>644</width>
      <height>644</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>ImageBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonShow">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示图形</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x的范围</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditXMin">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditXMax">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>y的范围</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditYMin">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditYMax">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>光标位置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>350</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditXCursor">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>350</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditYCursor">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>380</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReturn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <zorder>buttonShow</zorder>
   <zorder>imageBox</zorder>
   <zorder>label</zorder>
   <zorder>lineEditXMin</zorder>
   <zorder>lineEditXMax</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEditYMin</zorder>
   <zorder>lineEditYMax</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEditXCursor</zorder>
   <zorder>label_5</zorder>
   <zorder>lineEditYCursor</zorder>
   <zorder>buttonReturn</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MathPaint_Qt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonShow</sender>
   <signal>clicked()</signal>
   <receiver>MathPaint_QtClass</receiver>
   <slot>showClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>867</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>876</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showClicked()</slot>
 </slots>
</ui>
